### DISCUSSION TITLE
Only 6 digits appear in the UID field

### DISCUSSION THREAD

**Discussion by noamlivne** [chuot/rdio-scanner#155] [General]
With version 6.2.2 and SDRTrunk, In the Android app and browser, the UID field shows only the first 6 digits of unknown/unsaved UIDs.
Should be more digits in P25 UIDs.
This did not happen with 6.1.6 where more than 6 digits were shown.
UIDs that are saved in SDRTrunk are shown correctly.

**Comment by chuot** [comment]
I just tried and all the numbers are there on the wepapp and the native app, which is 7 in my case. Can you email me a MP3 file that has its UID truncated, I will test with it. Have you also checked the units section of your system in the admin dashboard, maybe the labels are wrong? It may also be related to the issue #150 where that feature was added to 6.2.0.

**Reply by noamlivne** [reply]
Hi Chrystian,
Thank you for looking into it.
Attached please find example recordings of UIDs that are not saved anywhere. Not in SDRTrunk nor in Rdio-scanner.
They should be 7 but appear as 6.
(UIDs that are saved in SDRTrunk Alias list, do seem to appear fine as 7 numbers, but without their name).

99.9% of my UIDs are not saved in SDRTrunk nor in Rdio-scanner. I do have Auto-populate.
I never looked at it before, but I see that I now have hundreds of strange Units saved as single digits. When opening them there are 6 digits UIDs instead of the needed 7. So perhaps there is a bug that involves the Auto-populate + UIDs?
[7nums appear as 6 nums.zip](https://github.com/chuot/rdio-scanner/files/8377834/7nums.appear.as.6.nums.zip)

**Reply by noamlivne** [reply]
And here is an image of the Units.
Hundreds of single digits as the Label (I have no idea where that came from - I do not save them in SDRTrunk nor Rdio-scanner - maybe from Auto-populate - but why single digit); And the ID is 6 numbers, when it is supposed to be 7 digits.
Perhaps the Auto-populate should not be applied on UIDs?

![image](https://user-images.githubusercontent.com/50629227/160781124-b72fe37e-e451-4619-8a71-23d0e56d14be.png)

**Comment by chuot** [comment]
It has been identified as a bug which is now fixed with v6.2.3. Unfortunately, you will have to clear all the buggy units in your system.

**Reply by noamlivne** [reply]
Thank you very much for the fix!
Works for me.

**Comment by chuot** [comment]
There was an error in the regex where I forgot to test for untagged units. I'm glad it works as it should. Thank you.
